src/components/ICOScanHeader.js

```diff
@@ -30,7 +28,7 @@ export const ICOScanHeader = ({ ...props }) => (
           <Row>
             <Col lg={5} md={1} sm={1} xs={1} />
             <Col lg={7} md={11} sm={11} xs={11} className="part transparency">
-              <p className="title added-by-person">Added by Person</p>
+              <p className="title added-by-person">Added by <b>{props.addedBy || 'Person'}</b></p>
```

Comment:
maybe if there is no `addedBy` specified we should hide whole paragraph? `Added by Person` looks weeeird.